src/javascript/lib/jquery.floatingscroll.js

```diff
@@ -0,0 +1,162 @@
+/*!
+ * jQuery floatingscroll Plugin 2.1.1
```

Comment:
Usually 'Do not add one' is a very good answer :)
These often are not well tested cross-browser by the dev, and often they stop supporting them.
So the 'fun' for doing the testing and then fixing remains on you.
Also, in general, adding a large component (this one is small) adds to loading times.

In this case for tables, there is a more complicated but better solution, and it is to reorganize the data so it is not exactly a table, and to reduce the info so it fits on the screen (this is for future versions, just sharing my thoughts :) )